app/views/api-keys/_key.njk

```diff
@@ -57,8 +57,7 @@
               }
             })
           }}
-          <a class="govuk-link govuk-link--no-visited-state
-govuk-link--no-visited-state target-to-show--cancel" href="#main">Cancel</a>
```

Comment:
Good point.

Using that class is not consistent with how the design system has marked up the cancel button:
https://design-system.service.gov.uk/components/button/ > Grouping buttons

`<a class="govuk-link" href="#">Cancel</a>`

Unless we really need to, I don't think we should use that CSS class.
Showing that a visited state maybe even be beneficial - indicating that a user has clicked that link before.

Therefore, I think we should remove it.

Reply:
The link is specific to an agreement. wouldn't it confuse if it is seen as clicked on a different agreement page?

Reply:
Just tested it locally by checking on different agreement detail pages.
Looks fine.

The link states only show up with JS switched off.

They are working because the links are to page fragments and append `#cancel-form` to the main URL.
Because the URLs for each agreement page are unique, it appears to work fine.

Please see screenshots below:

Agreement Page 1:
<img width="1453" alt="image" src="https://github.com/alphagov/pay-selfservice/assets/59831992/457a02f0-0a58-47d4-9fc5-3e7c3a0a4605">

Agreement page 2:
<img width="1453" alt="image" src="https://github.com/alphagov/pay-selfservice/assets/59831992/590835d1-d771-4731-bb28-c0c7a37cbb39">